langlib/langlib-test/src/test/resources/test-src/statements/foreach/foreach-var-type-negative.bal

```diff
@@ -0,0 +1,62 @@
+public type Foo record {|
```

Comment:
Shall we also add a test for an open (inclusive) record, with the default `anydata` rest field type?

```ballerina
type Baz record {
    string s;
    int i;
};
```

Reply:
sure